_targets.R

```diff
@@ -0,0 +1,32 @@
+library(targets)
source("code.R")
tar_option_set(packages = c("tidyverse", "sbtools", "whisker"))

```

Comment:
Here, you're pointing `targets` to the `code_.R` script as the source for each of the referenced functions, but your code is now broken into several phase-specific `.R` files instead of in a single `code.R` file, thanks to your re-organization work in the last issue. You should change this to point to each of those `.R` files instead, so that `targets` can find the functions that you've defined.

Reply:
^ ah I see you included those `source()` calls _within_ `code.R` -- apologies for missing that. We'd typically define them here, in multiple calls to `source()` so that anyone who uses the pipeline can see exactly which scripts are being referenced.

Reply:
Got it. I will take note of that.